### PR TITLE
Mdrogalis/overview

### PR DESCRIPTION
**Pull request by MichaelDrogalis**
### Description 

Updates the overview docs page to have strong nudges to helpful material. Eliminates duplication with microsite.

### CODE REVIEW

**Review by JimGalasyn — APPROVED** [review]
Very nice!

File: docs/index.md [hunk 1]
```diff
@@ -6,172 +6,94 @@ description: Learn about ksqlDB, the event streaming database for creating strea
...
+  </div>
+</div>
+
+## Learn
+
+<div class="cards">
+  <div class="card">
+    <strong>How it works</strong>
+    <ul class="card-items">
+      <li><a href="https://www.confluent.io/blog/how-real-time-stream-processing-works-with-ksqldb/">The SQL language</a></li>
```
Comment (colinhicks) [inline docs/index.md:50]: I feel like something at this hints more at the overview nature of the blog post. Looking at the text alone, I thought the link was going to be something like the syntax guide.
```suggestion
      <li><a href="https://www.confluent.io/blog/how-real-time-stream-processing-works-with-ksqldb/">Stream processing with SQL</a></li>
```

**Review by colinhicks — APPROVED** [review]
💯 Really nicely done! Had one suggestion for your consideration:

File: docs/index.md (hunk 1 above)
Comment (colinhicks) [reply docs/index.md:50]: fwiw, the suggestion still fits in the horizontal space
image

**Pull request merged**